pkg/kfapp/ossm/types.go

```diff
@@ -58,3 +68,28 @@ func (m *manifest) processTemplate(data interface{}) error {
 func ReplaceChar(s string, oldChar, newChar string) string {
 	return strings.ReplaceAll(s, oldChar, newChar)
 }
+
+func copyEmbeddedFS(fsys fs.FS, root, dest string) error {
```

Comment:
```suggestion
// In order to process the templates, we need to create a tmp directory
// to store the files. This is because embedded files are read only.
// copyEmbeddedFS ensures that files embedded using go:embed are populated
// to dest directory
func copyEmbeddedFS(fsys fs.FS, root, dest string) error {
```